api/pkg/apps/cms/casetype_validation.go

```diff
@@ -0,0 +1,21 @@
+package cms
+
+import "github.com/nrc-no/core/pkg/validation"
+
+func ValidateCaseType(caseType *CaseType, path *validation.Path) validation.ErrorList {
+	errList := validation.ErrorList{}
+
+	if len(caseType.Name) == 0 {
+		errList = append(errList, validation.Required(path.Child("name"), "name is required"))
+	} else if !validation.IsValidAlpha(caseType.Name) {
+		errList = append(errList, validation.Invalid(path.Child("name"), caseType.Name, "name should only contain letters and spaces"))
+	}
+	if len(caseType.PartyTypeID) == 0 {
```

Comment:
We should also make sure that all UUIDS are actually valid UUIDs
```
func IsValidUUID(str string) bool {
_, err := uuid.FromString(str);
return err == nil
}
```

Reply:
I assume that means adding a subroutine in the Status method to check for uuid errors and return a different error type in that case? The front-end user can't correct that type of error, right?

Is this really necessary though? All UUIDs are generated by the same library

Reply:
```
func ValidateUUID(uuid string, path validation.Path) validation.ErrList {...}
```